drivers/pn532/pn532.c

```diff
@@ -120,21 +120,19 @@ int pn532_init(pn532_t *dev, const pn532_params_t *params, pn532_mode_t mode)
     gpio_init(dev->conf->reset, GPIO_OUT);
     gpio_set(dev->conf->reset);
     dev->mode = mode;
-    if (mode == PN532_I2C) {
```

Comment:
Not sure of the fix here. This may increase the generated code size.

Reply:
Why would it?

Reply:
@aabadie If anything, the contrary is the case. The if is now in the `#ifdef`, so we get a condition check and 2 jump instructions less, even if the optimizer wouldn't already deal with that.